### ISSUE TITLE
Issue for me!

### ISSUE THREAD

**Issue by brandonlikesguns** [cohstats/coh2stats#149]
This site keeps deleting dozens of games I play constantly all of the time really wanting help to try and hopefully solve this issue! 
https://coh2stats.com/players/76561198800363045

**Comment by petrvecera** [comment]
Hi the player cards data are taken from Relic. We do not store this data. If there is issue with your player card - it has to be on the Relic side. 

As for the last matches. This is also taken from Relic - and Relic keeps only 10 matches for each mode. This means that only last 10 games for 4v4 is kept for you, if you play another games - it's deleted. 

We might be able to store /display some matches in future but for now it's not possible.

**Comment by brandonlikesguns** [comment]
I appreciate the reply. I understand it only keep records of the past 10
games for each mode but the overall amount of ranked games I have played in
total it keeps deleting dozens of them constantly. I notice it doesn't do
this for other players profiles just mine that I have noticed so far. On my
player profile I have gotten to around 1050 ranked games played very many
times and then it just deletes dozens overall around 50 to 60 games I have
played nonstop. One of my friends suggested maybe the sites counter or
something is messed up but I personally would like to know my real
statistics, Is there anybody that you suggest from Relics side I can
contact for help about this maybe!?
